### ISSUE TITLE
Null error when poster url not available

### ISSUE THREAD

**Issue by lucafluri** [lucafluri/ws6C-rating-tool#5]
Flutter raises an assertion error in the network_image_io package that the url is null.

This is raised when trying to open the detailed view of a movie without a poster.

**Comment by AndiSwiss** [comment]
This happens e.g. when search for  "howdy", then clicking on the first movie where no movie-poster was loaded

**Comment by AndiSwiss** [comment]
**Mostly fixed** -> see pull request #9 


**Status:**
It works in the app, doesn't throw anymore errors in the app and rating in the app works even for movies without a poster.

Though, there is still an error in the console when clicking on a movie-poster, which is empty:

```
════════ Exception caught by gesture ═══════════════════════════════════════════════════════════════
The following JsonUnsupportedObjectError was thrown while handling a gesture:
Converting object to an encodable object failed: Instance of 'Movie'

When the exception was thrown, this was the stack: 
#0      _JsonStringifier.writeObject (dart:convert/json.dart:659:7)
#1      _JsonStringStringifier.printOn (dart:convert/json.dart:846:17)
#2      _JsonStringStringifier.stringify (dart:convert/json.dart:831:5)
#3      JsonEncoder.convert (dart:convert/json.dart:260:30)
#4      JsonCodec.encode (dart:convert/json.dart:169:37)
...
Handler: "onTap"
Recognizer: TapGestureRecognizer#fbd11
  debugOwner: GestureDetector
  state: ready
  won arena
  finalPosition: Offset(124.2, 454.1)
  finalLocalPosition: Offset(124.2, 187.0)
  button: 1
  sent tap down
════════════════════════════════════════════════════════════════════════════════════════════════════

```
